app/controllers/users_controller.rb

```diff
@@ -113,6 +113,17 @@ def extlogout
     render :extlogout, :layout => 'login'
   end
 
+  def test_mail
+    begin
+      user = find_resource(:edit_users)
+      MailNotification[:tester].deliver(:user => user, :email => params[:user_email])
+    rescue => e
+      Foreman::Logging.exception("Unable to deliver email", e)
+      render :json => {:message => _("Sending email has failed, check logs for detailed info")}, :status => :unprocessable_entity and return
```

Comment:
"Unable to send email, check server logs for more information" might read better.  Consider putting the first line of the message in, or using a wrapped exception like the LDAP PR?